README.md

```diff
@@ -2,7 +2,7 @@
 
 This is the frontend application for the Probate Personal Applicants online service. The service provides a clear interface for citizens, presented as sequence of HTML 5 web pages designed to GDS Service Design guidelines, so that they can apply for Probate online. The service provides functionality for both single and multiple applicant journeys.
 
-The Frontend Application delegates a number of backend logic to the underlying services, including Persistence, Business and Submit services.
+The Frontend Application delegates a number of backend logic to the underlying services, including Orchestrator, Business and Submit services.
```

Comment:
This paragraph doesn't really make sense as the architecture has completely changed.
I think it should say it uses Orchestrator to route specific requests to the underlying services such as business service and submit service

Reply:
Done